packages/arb-provider-ethers/scripts/builder

```diff
@@ -1,8 +1,14 @@
 rm -r ./dist
 
-yarn run tsc
+tsc
```

Comment:
Will this work if typescript isn't installed globally? If so do we require a global installation?

Reply:
On second thought, I guess this is safe since `builder` is only run from the `package.json`?

Reply:
yep, it works without a global installation, I don't have one atm. though now I'm wondering if that's the reason that we used `yarn run` in the first place - to ensure the global installation isn't used. if the presence of a global installation is an issue, maybe this could be replaced with `npm run tsc` instead for the behavior we're looking for

Reply:
just tested it, doesn't conflict with global installations either

Reply:
Awesome